docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -714,6 +716,10 @@ _**default:**_ `application/xml+rss application/atom+xml application/javascript
 
 Enables or disables [HTTP/2](https://nginx.org/en/docs/http/ngx_http_v2_module.html) support in secure connections.
 
+## gzip-disable
+
+Disables gzipping of responses for requests with "User-Agent" header fields matching any of the specified regular expressions. The special mask `msie6` corresponds to the regular expression `MSIE [4-6]\.`, but works faster.
```

Comment:
Can you please add the same comment you did on description pointing to the reference in http://nginx.org/en/docs/http/ngx_http_gzip_module.html#gzip_disable ?

Thanks